dbms/src/Storages/FormatVersion.h

```diff
@@ -39,6 +39,7 @@ using Version = UInt32;
 inline static constexpr Version V0 = 0;
 inline static constexpr Version V1 = 1; // Add column stats
 inline static constexpr Version V2 = 2; // Add checksum and configuration
+inline static constexpr Version V3 = 3; // Use Meta V2
```

Comment:
Need to set STORAGE_FORMAT_V5.dm_file = DMFileFormat::V3;
 
https://github.com/pingcap/tiflash/blob/f327b7fd350def2719fb0c3b6ed30af290b1e9d4/dbms/src/Storages/FormatVersion.h#L124-L131